api/libproject/devstart.rb

```diff
@@ -1236,7 +1222,7 @@ def create_project_resources(gcc)
     common.run_inline("gcloud service-management enable #{service} --project #{gcc.project}")
   end
   common.status "Creating GCS bucket to store credentials..."
-  common.run_inline %W{gsutil mb -p #{gcc.project} -c regional -l us-central1 gs://#{gcc.project}-credentials/}
```

Comment:
Why is gsutil being removed here?

Reply:
Oops - not sure how that happened. Good catch.